frontend/src/Components/Login/Login.js

```diff
@@ -13,6 +14,16 @@ const Login = ({ history }) => {
         await app
           .auth()
           .signInWithEmailAndPassword(email.value, password.value);
+        var user = app.auth().currentUser;
```

Comment:
You should use `let` instead of `var` for variables. Here user value doesn't change so use `const`.